qiling/loader/pe.py

```diff
@@ -346,7 +380,10 @@ class QlLoaderPE(QlLoader, Process):
     def __init__(self, ql):
         super(QlLoaderPE, self).__init__(ql)
         self.ql         = ql
-        self.libcache   = self.ql.libcache
+        if type(self.ql.libcache) == bool:
```

Comment:
Specifying `True` for libcache still works, and it will use the default cache.
Specifying a class for `libcache` (or something callable),
it will directly use that as cache so the user can override it.